src/commcare_cloud/manage_commcare_cloud/changelog.md.j2

```diff
@@ -0,0 +1,17 @@
+# About this changelog
+
+The following changes to `commcare-cloud` that require your attention,
+newest first.
+
+Changes that will require an action from anyone choosing
+to update on or after the date listed will be maked "_action required_".
+Those which are worth taking a look at but which may or may not require
+an action on your part will be marked "_action optional_".
+
+
+## Changelog
+{% for changelog_entry in changelog_contents %}
```

Comment:
You can use `{%-` to mean trim whitespace (the newline) before and `-%}` to mean trim whitespace after. You can use these two things together to address unwanted whitespace in the output.

Reply:
https://github.com/dimagi/commcare-cloud/pull/1979/commits/a516bc3342f8e0bded4bfa2bb1e07dba8169d99b

Reply:
https://github.com/dimagi/commcare-cloud/pull/1979/commits/dc835ad45ce608c7b6948272a3195f999da72319